packages/app-cli/tests/models_Tag.ts

```diff
@@ -123,21 +133,25 @@ describe('models_Tag', function() {
 		expect(commonTags.length).toBe(0);
 
 		commonTags = await Tag.commonTagsByNoteIds([note0.id, note1.id, note2.id, note3.id]);
+
 		let commonTagIds = commonTags.map(t => t.id);
 		expect(commonTagIds.length).toBe(0);
 
 		commonTags = await Tag.commonTagsByNoteIds([note1.id, note2.id, note3.id]);
+
 		commonTagIds = commonTags.map(t => t.id);
 		expect(commonTagIds.length).toBe(1);
 		expect(commonTagIds.includes(taga.id)).toBe(true);
 
 		commonTags = await Tag.commonTagsByNoteIds([note2.id, note3.id]);
+
 		commonTagIds = commonTags.map(t => t.id);
 		expect(commonTagIds.length).toBe(2);
 		expect(commonTagIds.includes(taga.id)).toBe(true);
 		expect(commonTagIds.includes(tagb.id)).toBe(true);
 
 		commonTags = await Tag.commonTagsByNoteIds([note3.id]);
+
 		commonTagIds = commonTags.map(t => t.id);
```

Comment:
It's a small thing, but you could you remove the white space changes please? Just to keep the diff and blame history simpler.

Reply:
It's done